pkgs/servers/urserver/default.nix

```diff
@@ -9,11 +9,11 @@
 
 stdenv.mkDerivation rec {
   pname = "urserver";
-  version = "3.6.0.745";
+  version = "3.9.0.2465";
 
   src = fetchurl {
-    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/745/urserver-${version}.tar.gz";
-    sha256 = "1ib9317bg9n4knwnlbrn1wfkyrjalj8js3a6h7zlcl8h8xc0szc8";
+    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/${builtins.elemAt (builtins.splitVersion "${version}") 3}/urserver-${version}.tar.gz";
```

Comment:
```suggestion
    url = "https://www.unifiedremote.com/static/builds/server/linux-x64/${elemAt (splitVersion version) 3}/urserver-${version}.tar.gz";
```

Does that work?

Reply:
I get `undefined variable 'elemAt'`?

Reply:
I thought they where globally enabled. Not sure if builtins or lib is prefered.

Reply:
Don't know if there's a policy, but some statistics from the nixpkgs repo:
```
$ git grep -o builtins.elemAt | wc -l
53
$ git grep -o lib.elemAt | wc -l
36
$ git grep -o builtins.splitVersion | wc -l
11
$ git grep -o lib.splitVersion | wc -l
8
```
Looks like `builtins` is used slightly more often, but it's not very decisive.